lib/ovirt_metrics/models/ovirt_history.rb

```diff
@@ -1,8 +1,17 @@
 module OvirtMetrics
   class OvirtHistory < ActiveRecord::Base
+    attr_writer :connection_specification_name if ActiveRecord::VERSION::MAJOR < 5
+
     self.abstract_class = true
     self.pluralize_table_names = false
 
+    def self.connection_specification_name
+      if !defined?(@connection_specification_name) || @connection_specification_name.nil?
+        return self == OvirtHistory ? OvirtMetrics.config.connection_specification_name : superclass.connection_specification_name
+      end
```

Comment:
Yup you betcha. This needs to be here as it's written there because we're basically just trying to do the same thing with a different pool independent of Base, one step higher in the hierarchy. Without it, descendants of OvirtHistory will constantly try and grab the wrong connection name from the config.